mne/time_frequency/tfr.py

```diff
@@ -215,8 +215,7 @@ def _cwt(X, Ws, mode="same", decim=1, use_fft=True):
 
     # Decimating is performed after centering the convolution, and can
     # therefore lead to 1 time sample jittering.
-    jitter = (n_times % decim) % 2
-    n_times_out = n_times // decim + jitter
+    n_times_out = int(np.ceil(float(n_times) / decim))
```

Comment:
Casting to `float`, doing inexact computations, and going back to `int` is less clean than using exact integer arithmetic (divide and modulo). Can you do this with integer arithmetic?